resources/test/fixtures/flake8_simplify/SIM300.py

```diff
@@ -0,0 +1,9 @@
+# Bad
+
+'yoda' == compare # SIM300
+42 == age # SIM300
```

Comment:
One condition that `flake8-simplify` seems to get wrong:

```py
"yoda" == "yoda"
```

`flake8-simplify` gives me:

```
resources/test/fixtures/flake8_simplify/SIM300.py:10:1: SIM300 Use '"yoda" == "yoda"' instead of '"yoda" == "yoda"' (Yoda-conditions)
```

But we should probably avoid raising this in the event that both sides are constant?

Reply:
Oh good catch, I'll add another check. Might be worth creating an alert at some point when comparing constants with each other too. Can't think of any good reason to do it on purpose.

Reply:
Haha yeah the absurdity of that didn't even occur to me... I have my correctness hat on.